tests/e2e/__testers__.ts

```diff
@@ -4,63 +4,167 @@ import { values } from 'lodash'
 import stripAnsi from 'strip-ansi'
 import { ClientBase } from '@prisma-spectrum/reflector/dist-cjs/Client'
 import { Reflector } from '@prisma-spectrum/reflector'
+import { PrismaClient } from '@prisma/client'
+import { log } from 'floggy'
+import { PrismaClientConstructor } from '~/tests/e2e/helpers/getMysqlAdminPrismaClient'
+import { casesHandled } from '~/src/utils'
 
-export const testTemplate = (params: {
+export interface DBTestParams {
   templateName: PrismaTemplates.$Types.Template['_tag']
-  /** A string or regex pattern of the expected output (stdout) when the project dev mode is run.*/
   expectedDevOutput: RegExp | string
-}) => {
-  jest.setTimeout(50_000)
+  datasourceProvider: Reflector.Schema.DatasourceProviderNormalized
+  prismaConfig?: {
```

Comment:
This should be `templateConfig` should it not? And it should be `Pick<TemplateConfig, 'referentialIntegrity'>` right? 

If this is too much trouble right now (because I'm not 100% sure for example how to get `TemplateConfig` off the top of my head) please just leave a TODO.

Reply:
I renamed the prop and I think I found the interface. Is it the correct one? 
```
Pick<PrismaTemplates.$Types.BaseTemplateParameters, 'referentialIntegrity'>
```

Reply:
Yep looks right!